### DISCUSSION TITLE
Adjust difficulty mutiplier for DT

### DISCUSSION THREAD

**Discussion by cuhoangdk** [ppy/osu#21914] [Ideas] [answered]
Use this functon
for DT/NC: `difficulty mutiplier=(0.12/0.5)*(speed increase -1)+1`
I haven't found the function for HT/DC yet but i hope this can help.

**Comment by frenzibyte** [comment, marked as answer]
What you've provided is generally referred to as a linear scale function, and what it indicates is that difficulty is linearly increased as the speed increases, which isn't quite the case. This is tracked in https://github.com/ppy/osu/discussions/19711 with a better function.
